model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -1045,10 +1059,7 @@ def model_output_to_completion_output(
             tokens = None
             if with_token_probs:
                 tokens = [
-                    TokenOutput(
-                        token=model_output["tokens"][index],
-                        log_prob=list(t.values())[0],
-                    )
+                    TokenOutput(token=model_output["tokens"][index], log_prob=list(t.values())[0])
```

Comment:
only comment here is - can you run the pre-commit hooks? I don't think these files should be changing. Or it could be possible that the old files didn't conform to pre-commit, would you mind checking?

Reply:
hm I ran `black.toml` and it left my code changes unchanged so not really sure if I should run anything else?